src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -420,8 +467,37 @@ export class BaseEditCourseForm extends React.Component {
               className="course-image"
               disabled={disabled}
             />
-            {showMarketingFields
-            && (
+            <Field
+              name="tags"
+              itemType="topic"
+              component={RenderInputTextField}
+              fetchSuggestions={fetchCourseTagSuggestions(allCourseTags)}
+              renderSuggestion={renderCourseTagSuggestion}
+              type="text"
+              label={(
+                <>
+                  <FieldLabel
+                    id="tags.label"
+                    text="Topics"
+                    helpText={(
+                      <p>
+                        You can add comma separated tags
+                        <br />
+                        Format: mba-no-gmat,mba,mba_4_modules,mba_NY
+                      </p>
+                    )}
+                    optional
+                  />
+                  {courseTags
+                    && courseTags.length ? this.formatCourseTags(courseTags, setCourseTags) : null}
+                </>
+              )}
+              disabled={disabled || !administrator}
```

Comment:
nit: administrator equals is_staff. The course editors might not have is_staff true for them. We can double check if editors should have perms to add the tags.